plugins/defaultplugins/ifplugin/interface_config.go

```diff
@@ -248,6 +252,35 @@ func (plugin *InterfaceConfigurator) ConfigureVPPInterface(iface *intf.Interface
 	return wasError
 }
 
+/**
+	Set rx-mode on specified VPP interface
+ */
+func (plugin *InterfaceConfigurator) configRxModeForInterface(iface *intf.Interfaces_Interface, ifIdx uint32) error {
+	rxModeSettings := iface.RxModeSettings
+	if rxModeSettings != nil {
+		switch iface.Type {
+		case intf.InterfaceType_ETHERNET_CSMACD:
+			if rxModeSettings.RxMode == intf.RxModeType_POLLING {
+				return plugin.configRxMode(iface, ifIdx, *rxModeSettings)
+			}
+		default:
+			return plugin.configRxMode(iface, ifIdx, *rxModeSettings)
+		}
+	}
+	return nil
+}
+
+/**
+	Call concrete vpp API method for setting rx-mode
+ */
+func (plugin *InterfaceConfigurator) configRxMode(iface *intf.Interfaces_Interface, ifIdx uint32, rxModeSettings intf.Interfaces_Interface_RxModeSettings) error {
+	err := vppcalls.SetRxMode(ifIdx, rxModeSettings, plugin.Log, plugin.vppCh, nil)
```

Comment:
add stopwatch (instead of hardcoded nil)

measure.GetTimeLog(SwInterfaceSetRxMode{}, plugin.Stopwatch)

also in update/delete

Reply:
is it also necessary to do delete of rx-mode?